phpstan.neon

```diff
@@ -40,6 +40,7 @@ parameters:
         - %currentWorkingDirectory%/src/Component/vendor/*
 
     ignoreErrors:
+        - '/Call to method type\(\) on an unknown class Doctrine\\ORM\\Mapping\\AssociationMapping./'
```

Comment:
What about using phpstan-doctrine here?

```shell
composer require --dev phpstan/phpstan-doctrine
```

Reply:
no, I don't think that's the solution. 
This class does not exist on doctrine orm 2.